test_bucketlist/test_bucket_list.py

```diff
@@ -0,0 +1,72 @@
+import os
+import json
+from run_app import app, db
+
+from base_test import BaseTestCase
+from models import User, BucketList, Item
+
+URL = "/api/v1.0/bucketlists/"
+
+class BucketListTestCase(BaseTestCase):
+    """ Test suite for BucketList API """
+    def setUp(self):
+        """ Create resources to get auth token for login and access """
+        db.create_all()
+
+        new_user = User(
+            username='newguy',
+            email_address="test@user.com"
+        )
+        new_user.hash_this_pass("thispass")
+        db.session.add(new_user)
+        db.session.commit()
+
+        data = json.dumps(dict(
+            email="test@user.com",
+            password="thispass"
+        ))
+
+        response = self.client.post("/api/v1.0/auth/login/", \
+        data=data, content_type="application/json")
+        data = json.loads(response.data.decode())
+        auth_token = data['auth_token']
+
+        self.headers = {
+            'Authorization':'Token ' + auth_token,
+            'Content-Type': 'application/json',
+            'Accept':'application/json',
+        }
+
```

Comment:
There is code repetition here since you have `setUp` and `tearDown` methods in base class.